src/edu/csus/ecs/pc2/clics/API202003/ClarificationService.java

```diff
@@ -51,7 +50,7 @@ public ClarificationService(IInternalContest inContest, IInternalController inCo
     /**
      * This method returns a representation of the current contest groups in JSON format. The returned value is a JSON array with one language description per array element, matching the description
      * at {@link https://clics.ecs.baylor.edu/index.php/Draft_CCS_REST_interface#GET_baseurl.2Flanguages}.
```

Comment:
Again, nothing to do with this PR, but any link to "baylor.edu" is just wrong...

Reply:
There are no fewer than a dozen more places where Baylor is mentioned throughout the code base.  The 202003 package is the original stuff (just relocated).  Should we go through and change all those Baylor's? (IE delete them?)  There is none of that in the new 202306 package.

Reply:
I guess that "spending time going through that old code just to fix those bad 'Baylor' references" probably falls into "Bigger Fish To Fry"...  so I think we should fix them IF we come across them while making other changes, but otherwise... well, "BFTF".